.circleci/config.yml

```diff
@@ -1026,7 +1030,7 @@ jobs:
     <<: *base_ubuntu2404_clang
     environment:
       <<: *base_ubuntu2404_clang_env
-      CMAKE_OPTIONS: << parameters.cmake_options >>
+      CMAKE_OPTIONS: -DPEDANTIC=OFF << parameters.cmake_options >>
```

Comment:
given the size of this file, perhaps add a comment here as well that could say, e.g., `# NOTE: Disabled pedantic builds to avoid false positives, see b_ubu_asan for a more detailed explanation`